src/utils/sendMail.js

```
@@ -0,0 +1,29 @@
+const nodemailer = require('nodemailer');
+
+require('dotenv').config();
+
+// eslint-disable-next-line no-console
+console.log('SMTP HOST:', process.env.SMTP_HOST);
+
+const sendMail = async (to, subject, html) => {
+  const transporter = nodemailer.createTransport({
```

The function creates a new Nodemailer transporter on every call. Creating and verifying a transporter is relatively expensive. Create the transporter once (module-level) and reuse it for subsequent sends. Also allow creating the transporter asynchronously at startup so you can verify connection and fail fast if SMTP config is invalid.